clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress

```diff
@@ -52,15 +54,17 @@ class SIPpstats(csv.excel):
 SIPP_BIN = "/usr/share/clearwater/bin/sipp_coreonly"
 SCRIPTS_DIR = "/usr/share/clearwater/sip-stress/"
 
-REGISTER_CSV = "/tmp/registers.csv"
-INVITE_CSV = "/tmp/calls.csv"
+REGISTER_CSV = NamedTemporaryFile(prefix="sipp_register")
+INVITE_CSV = NamedTemporaryFile(prefix="sipp_calls")
```

Comment:
I want that behaviour, actually - we don't want these CSV files cluttering up the disk, so using a NamedTemporaryFile this way means they get automatically deleted when the filehandles are closed at program exit.